CONTRIBUTING.md

```diff
@@ -41,6 +41,14 @@ return it. Once we receive it, we'll be able to accept your pull requests.
 
 ***NOTE***: Only original source code from you and other people that have signed the CLA can be accepted into the main repository.
 
+### Setting up an environment
+1. Install [pyenv](https://github.com/pyenv/pyenv#installation)
+2. Create a 3.9.9 virtual env `PYENV_VERSION=3.9.9 pyenv exec python -m venv venv`
```

Comment:
This reads as if it is mandating 3.9.9, but I think it's just an example.  Do we somewhere specify the minimum Python version?  Perhaps "Create a virtual environment.  For example, `...`"?

You removed the `pyenv install`, which is strange since you left the "Install pyenv" step.  Perhaps "Install the minimum Python version.  For example, `pyenv install 3.9.9`"?

I believe this will create a `venv` directory.  Should we add `venv/` to `.gitignore`?